test/module/libs/crypto/keys_manager_test.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+#include <fstream>
+#include <string>
+#include "crypto/keys_manager_impl.hpp"
+
+#define PUBKEY_SIZE 32
+#define PRIVKEY_SIZE 32
+
+using namespace iroha;
+using namespace boost::filesystem;
+using namespace std::string_literals;
+
+class KeyManager : public ::testing::Test {
+ public:
+  bool create_file(const path &ph, const std::string &contents) {
+    std::ofstream f(ph.c_str(), std::ifstream::binary);
```

Comment:
Why `ifstream::binary` instead of `ofstream::binary` ?

Reply:
redundant line actually, thanks for noticing